src/libs/API.js

```diff
@@ -547,7 +547,7 @@ function ResendValidateCode(parameters) {
  */
 function SetPassword(parameters) {
```

Comment:
you might want to update the params description of this method to capture parameters.accountID